std/experimental/allocator/package.d

```diff
@@ -3323,8 +3323,6 @@ nothrow:
     theAllocator.dispose(arr);
 }
 
-__EOF__
```

Comment:
LOL. What was the point of it? Was this an accident?

Reply:
Probably, it was introduced at the end of the file just before a `version(none)` declaration.

Reply:
https://github.com/dlang/phobos/blob/8b249a6240fe9a923da69c7122fc27059640ff59/std/allocator.d#L3987

Reply:
I just was about to create a PR to remove this.

I found it in my attempts to refactor phobos for phobos v2, my refactoring didn't do well with this. @MoonlightSentinel was on the beerconf call when I found it, thanks for fixing it!

> Was this an accident?

I actually emailed Andrei, and he said he remembered some compiler bug making him put something like that in.